.github/workflows/fedora-copr.yml

```diff
@@ -0,0 +1,37 @@
+name: Package for Fedora Copr repo
+
+on:
+  release:
+    types: [published]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  package:
+    container: fedora:latest
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install API token for copr-cli
+        env:
+          API_TOKEN_CONTENT: ${{ secrets.COPR_API_TOKEN }}
```

Comment:
Add a comment above `# To generate a new token: https://copr.fedorainfracloud.org/api/`. I'm kinda forgetful...

Reply:
Done